### PR TITLE
I'll show myself out

### PR DESCRIPTION
**Pull request by dmlloyd**

### CODE REVIEW

File: implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
     static final Comparator<ConfigSource> CONFIG_SOURCE_COMPARATOR = new Comparator<ConfigSource>() {
         @Override
         public int compare(ConfigSource o1, ConfigSource o2) {
-            int res = Long.signum((long) o2.getOrdinal() - (long) o1.getOrdinal());
+            int res = Integer.compare(o2.getOrdinal(), o1.getOrdinal());
```
Comment (jmesnil) [inline implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java]: Is it correct? (do we have a test for config source ordering?)

It seems strange to me that the method parameters for `Comparator#compare` are `o1, o2` but the parameters for `Integer.compare` are `o2, o1`. I'd have expected parameters to be in the same order for both

Comment (jmesnil) [reply implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java]: never mind, the comparison order for `ConfigSource` is determined by the *highest* ordinal so it's normal that parameters order is inverted.

**Comment by kenfinnigan** [comment]
Argh, let me re-add properties I removed for testing Jacoco stuff so this passes

File: implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java (hunk 1 above)
Comment (dmlloyd) [reply implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java]: Yes there is a test for it.  I initially had it backwards :)

**Pull request merged**